src/software/ai/evaluation/time_to_travel.cpp

```diff
@@ -0,0 +1,74 @@
+#include "software/ai/evaluation/time_to_travel.h"
+
+#include <cmath>
+
+Duration getTimeToTravelDistance(const double distance, const double max_velocity,
+                                 const double max_acceleration,
+                                 const double initial_velocity,
+                                 const double final_velocity)
+{
+    // Bound all values to be realistic
+    double d_total = std::max(0.0, distance);
+    double v_max   = std::max(0.0, max_velocity);
```

Comment:
this rejection of negative should be part of the function comments

Reply:
Added it to the javadoc